.eslintrc.json

```diff
@@ -5,24 +5,24 @@
     },
     "extends": [
         "plugin:react/recommended",
-        "standard-with-typescript",
-        "plugin:@typescript-eslint/recommended",
-        "eslint:recommended"
-    ],
-    "overrides": [
+        "airbnb",
```

Comment:
Generally it is a nice habit to create "atomic" PRs. If these chores e.g. editing configs are related to your current task, include it in the PR, but otherwise it is easier to review if you split it into a separate PR.

Reply:
Agreed, it is only lazyness from my part